### DISCUSSION TITLE
The "GNU" assemly

### DISCUSSION THREAD

**Discussion by shenlebantongying** [rms-support-letter/rms-support-letter.github.io#6778] [General]
They created a gnu.org clone and they are not the GNU you know
https://gnu.tools/

Not sure if they try to split the community, or fake a GNU, or try to take over GNU, or they just want to try a new approach.

Current committee members (since Jan. 2020)

- Andy Wingo -> igalia
- Carlos O'Donell -> Red Hat
- Mark Wielaard -> Red Hat
- Ludovic Courtès -> INRIA
- Andreas Enge -> INRIA

Their emailing list:
The assembly’s position -> https://lists.gnu.tools/hyperkitty/list/assembly@lists.gnu.tools/thread/SMFKD7M34VUTUW45MSO4UOWL4C7V5FQT/
Web site goes public tomorrow -> https://lists.gnu.tools/hyperkitty/list/assembly@lists.gnu.tools/thread/O64CDB5ACX2FVFHUTEKODOLVACAIAKPW/
Welcome to the GNU Assembly -> https://lists.gnu.tools/hyperkitty/list/assembly@lists.gnu.tools/thread/OWDCIZNUOCGLGBT4LKLJXMJCCLZ3DMZN/

**Comment by 6r1d** [comment]
> Not sure if they try to split the community, or fake a GNU, or try to take over GNU, or they just want to try a new approach.

Well, their intro explains quite a bit, doesn't it? It's clearly not made with **checking acusations** and **being neutral** in mind.

> To me, it’s a betrayal, as if the forced resignation two years ago was a masquerade.
